docs/backend/Coding-Guidelines/exception-handling.md

```diff
@@ -6,12 +6,14 @@ We separate our business exceptions from technical exceptions. While for technic
 
 By default, implementations of BusinessException must define
 
-```JSON
-code: 500
-type: "CUSTOM_ERROR_TYPE",
-title: "Custom Error Type",
-message: "Human readable details",
-// additional: optionalData
+```typescript
+const example = {
+  code: 500,
+  type: "CUSTOM_ERROR_TYPE",
+  title: "Custom Error Type",
+  message: "Human readable details"
+  // additional: optionalData
+}
```

Comment:
Why has this been changed from `JSON` to `typescript`?

Reply:
I think this information is outdated. Maybe we leave it like it was and nark it as `typescript`.